packages/contracts-bedrock/deploy-config/hardhat.json

```diff
@@ -53,8 +53,8 @@
   "faultGameWithdrawalDelay": 302400,
   "preimageOracleMinProposalSize": 126000,
   "preimageOracleChallengePeriod": 86400,
-  "proofMaturityDelaySeconds": 12,
-  "disputeGameFinalityDelaySeconds": 6,
+  "proofMaturityDelaySeconds": 604800,
+  "disputeGameFinalityDelaySeconds": 302400,
```

Comment:
This diff is required to avoid issues with the upgrade tests. Without this, the implementations will differ and we get an issue with StandardValidator. 

@mds1 we can apply this diff to the proposal branch to avoid that awkward upgrade of the ASR contract if we care to do it.